### ISSUE TITLE
A Full Local Deployment of the Whole Project

### ISSUE THREAD

**Issue by Mohido** [ELTE-Clean/Clean-Administration-Platform#3]
At the moment we only have the databases and their management gui systems firing up in the docker-compose. However we need to also include the backend and the front end to the docker-compose environment. For this issue, a new folder that contain the built code (Transpiled or whatsoever) of the backend and frontend. The backend needs to be transpiled with babel and currently it is built into a folder called dist/ in the backend folder. If we have a server side rendering application then we need the backend and the frontend to be built in a single folder for the ease of usage. 

To summarize, we need to fully deploy and fire-up the whole project through 1 docker-compose network. And at the moment the docker-compose file only deploys the databases and their GUI managers.

**Comment by the4t4** [comment]
Can be closed.
